build.gradle

```diff
@@ -122,6 +122,12 @@ jacocoTestReport {
   }
 }
 
+test {
+  testLogging {
+    showStandardStreams = true
```

Comment:
What is this for?

Reply:
If this is not set, anything that goes to standard outout (like System.out or logging) or standard err is not shown in the console during `gradle test`.

Maybe IntelliJ changes this internally so that you can always see logging output, but other environments don't. Note that the full logs still are available in a html report when you run `gradle test` without this, but its annoying to look those up.